pkg/services/hivelocity/hvtag/hvtag.go

```diff
@@ -182,3 +187,36 @@ func (deviceTag DeviceTag) RemoveFromList(tagList []string) (newTagList []string
 	}
 	return newTagList, updated
 }
+
+// IsEphemeralTag returns False if a tag should not get removed
+// if a machine leaves a cluster.
+// tag: Something like "caphv-cluster-name=hv-guettli"
+func IsEphemeralTag(tag string) bool {
+	if !strings.HasPrefix(tag, "caphv-") {
+		return false
+	}
+	for _, keepPrefix := range []string{
```

Comment:
comment:
// ignore tags that are only allowed to be changed or removed by the user